go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/google/btree
```

Comment:
not sure what the syntax is for 'mod' files, but can you add a copyright notice up here like the .go files?

Reply:
It appears comments play nice, so I moved the copyright and license section from other the files; happy to fix it up, if you want different wording.

Reply:
I would suggest adding a blank line to separate the copyright notice from the rest of the go.mod file content.

There is a blank line separating the copyright notice from the rest of .go files, e.g., see https://github.com/google/btree/blob/bfb38958ad2298964492841280eea188b9c23e7b/btree_test.go#L14 and https://github.com/google/go-cloud/blob/6482f3715c170669214c5cf4ba8421c51d772d30/go.mod#L14.

Reply:
see #31